apps/web/src/ui/swap/cross-chain/cross-chain-swap-trade-review-dialog.tsx

```diff
@@ -315,8 +337,11 @@ export const CrossChainSwapTradeReviewDialog: FC<{ children: ReactNode }> = ({
     },
   })
 
+  // Speeds up typechecking in the useMemo below
+  const _simulation: { request: any } | undefined = simulation
+
   const write = useMemo(() => {
-    if (!simulation) return undefined
+    if (!_simulation?.request) return undefined
```

Comment:
is this a better check?

Reply:
it's pretty much the same, was just bringing it in line with other files where it's simulation?.request